### PR TITLE
Update flu season language

### PR DESCRIPTION
**Pull request by joverlee521**
Describes the progress of the current flu season more accurately by saying "we are X months and X weeks into the 2019-2020 flu season".

This is just a quick patch that will remove the language that people have been unsure about (e.g. "we are about one-third of the way through flu season"). 

In the future we are going to remove references to "flu season" completely, and just show the "flu circulation".

### CODE REVIEW

File: current/src/components/CurrentConditions.js [hunk 1]
```diff
@@ -30,22 +30,11 @@ export default class CurrentConditions extends React.Component {
...
-      2: 'about two-thirds of the way through',
-      3: 'over two-thirds of the way through',
-      4: 'nearing the end of',
-      5: 'finishing up',
-      6: 'done with',
-      7: 'done with',
-      8: 'done with',
-    };
+    const sinceSeasonStart = currentDate.diff(FLU_SEASON_START_DATE, ['months', 'weeks']).toObject();
+    const monthsSinceSeasonStart = (sinceSeasonStart.months);
```
Comment (kairstenfay) [inline current/src/components/CurrentConditions.js]: These parentheses can be removed

File: current/src/components/CurrentConditions.js [hunk 2]
```diff
@@ -30,22 +30,11 @@ export default class CurrentConditions extends React.Component {
...
+    const weeksSinceSeasonStart = Math.round(sinceSeasonStart.weeks);
```
Comment (kairstenfay) [inline current/src/components/CurrentConditions.js]: Do we want to use `floor` or `ceil` instead of `round`?

Comment (joverlee521) [reply current/src/components/CurrentConditions.js]: I had initially tried to use `floor` which resulted in
1. Week 51 showing "we are 4 weeks into..."
2. Week 52 showing "we are 1 month into..." 

This didn't make sense to me. I think it should go from "4 weeks" to "1 month and 1 week", which is the behavior using `round`

Comment (kairstenfay) [reply current/src/components/CurrentConditions.js]: Fair enough.

**Comment by kairstenfay** [comment]
I think we should go ahead and merge/deploy this PR.

**Comment by kairstenfay** [comment]
[Describe progress of current flu season more accurately](https://trello.com/c/0yGyRD7m/283-describe-progress-of-current-flu-season-more-accurately)

**Pull request merged**